### PR TITLE
#6168: Filter feature selection in feature grid is disabled when advanced search is closed

### PR DESCRIPTION
**Pull request by AGMETEOR**
## Description

This PR fixes the issue where when the Advanced Search is opened > then feature editor gets automatically closed > then Advanced Search closed, the map can nolonger handle clicks that were added by the `handleClickOnMap` epic

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#6168 

**What is the new behavior?**

The map still handles clicks that update filter even when Advanced Search is opened and closed.

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/epics/featuregrid.js [hunk 1]
```diff
@@ -445,7 +445,8 @@ export const handleClickOnMap = (action$, store) =>
             })
                 .takeUntil(Rx.Observable.merge(
                     action$.ofType(UPDATE_FILTER).filter(({update = {}}) => update.type === 'geometry' && !update.enabled),
-                    action$.ofType(CLOSE_FEATURE_GRID, LOCATION_CHANGE)
+                    // action$.switchMap(() => Rx.Observable.zip([action$.ofType(CLOSE_FEATURE_GRID_CONFIRM), action$.ofType(CLOSE_FEATURE_GRID)])),
```
Comment (MV88) [inline web/client/epics/featuregrid.js:448]: please remove this comment

**Review by MV88 — CHANGES_REQUESTED** [review]
it seems the click is not working yet
can you check again ?

6168 not resolved

**Comment by AGMETEOR** [comment]
> it seems the click is not working yet
> can you check again ?
> 
> 6168 not resolved

@MV88 I can confirm it works. Please check again.

**Review by MV88 — APPROVED** [review]
apparently a fail to pull the changes made me testing the old code. 

actually it is indeed working. so i'll approve

**Pull request merged**